uid-pages/page-admin-process-list/src/resources/page.properties

```diff
@@ -0,0 +1,5 @@
+displayName=Bonita Admin Process List
+name=custompage_AdminProcessListBonita
+description=Page allowing to list all the processes for admin profile
+resources=[GET|bpm/process]
```

Comment:
I guess some resources need to be added (like the PUT for the state and the get for the user)

Reply:
There is no GET for the user, it is a deploy, but for the PUT, I'll add it